packages/docs/src/index.css

```diff
@@ -1,71 +1,3 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-
-@layer components {
-  /* Docs styling */
-  .blox-code-editor {
-    @apply rounded-lg mb-4 font-mono text-sm bg-gray-900 text-gray-300;
```

Comment:
This what makes everything themeable , so when after you run `create-blox-app` you'll be able to edit the index.css for your liking.

Reply:
class `blox-code-editor` doesn't exists anymore,
I removed all the name classes and all the tailwind.
so I am not sure 
do you want to keep empty index.css ?